internal/api/keppel/manifests_test.go

```diff
@@ -43,18 +43,8 @@ func TestManifestsAPI(t *testing.T) {
 		h := s.Handler
 
 		// setup two test accounts
-		mustInsert(t, s.DB, &models.Account{
-			Name:                     "test1",
-			AuthTenantID:             "tenant1",
-			GCPoliciesJSON:           "[]",
-			SecurityScanPoliciesJSON: "[]",
-		})
-		mustInsert(t, s.DB, &models.Account{
-			Name:                     "test2",
-			AuthTenantID:             "tenant2",
-			GCPoliciesJSON:           "[]",
-			SecurityScanPoliciesJSON: "[]",
-		})
+		test.MustExec(t, s.DB, `INSERT INTO accounts (name, auth_tenant_id) VALUES ('test1', 'tenant1')`)
+		test.MustExec(t, s.DB, `INSERT INTO accounts (name, auth_tenant_id) VALUES ('test2', 'tenant2')`)
```

Comment:
We can also use `test.WithAccount()` for this, no? That appears to have the logic to default the `PoliciesJSON` fields to `[]`, too.